src/app/services/theme/theme.service.ts

```diff
@@ -0,0 +1,202 @@
+import { DOCUMENT } from "@angular/common";
+import { Inject, Injectable } from "@angular/core";
+import { IS_SERVER_PLATFORM } from "src/app/app.helper";
+
+export const bawThemes = [
+  "highlight",
+  "primary",
+  "secondary",
+  "success",
+  "info",
+  "warning",
+  "danger",
+  "light",
+  "dark",
+] as const;
+
+export const bawThemeVariants = [
+  "",
+  "-lighter",
+  "-lightest",
+  "-darker",
+  "-darkest",
+] as const;
+
+export type BawTheme = typeof bawThemes[number];
+export type BawThemeVariant = typeof bawThemeVariants[number];
+
+export interface RGB {
```

Comment:
Conventionally such a value would be called Rgb24

Reply:
Could you potentially link to some relevant examples? I can't seem to find much on this convention, and as it will likely apply to a lot of the data I'm dealing with, it'd be nice to use standard names.

Reply:
I am of two minds on this, because it is not strictly true, unless we were using a UInt8 clamped array 

Anyway: web versions: https://github.com/Automattic/node-canvas#image-pixel-formats-experimental